client/package.json

```diff
@@ -41,11 +49,16 @@
         "eslint-plugin-react-compat": "0.0.3",
         "if-env": "^1.0.0",
         "ignore-styles": "^5.0.1",
+        "jest": "^22.4.2",
+        "jest-css-modules": "^1.1.0",
         "mocha": "^5.0.0",
         "preact-cli": "^2.0.1",
         "preact-cli-sw-precache": "^1.0.3",
+        "preact-compat-enzyme": "^0.2.5",
         "preact-jsx-chai": "^2.2.1",
```

Comment:
I guess this could probably go too since using jest

Reply:
I kept preact-compat-enzyme as jest is using that to render the DOM. I could get rid of the actual "enzyme" package though.

Reply:
I was referring to  preact-jsx-chai here, probably mocha and chai could go at this point too

Reply:
Ah gotcha. I grabbed the merge you just did and then took out "chai", "mocha" and "preact-jsx-chai" and the tests ran successfully.